networks/known_networks.go

```diff
@@ -637,25 +638,52 @@ func MustGetSelectedNetworksFromEnv() []blockchain.EVMNetwork {
 			wsEnvVar := fmt.Sprintf("%s_URLS", networkKeys[i])
 			wsEnvVal := os.Getenv(wsEnvVar)
 			if wsEnvVal == "" {
-				panic(errors.Errorf("set %s env var", wsEnvVar))
+				// Get default value
```

Comment:
Worth a warning log to indicate this is what you're doing, something like `log.Warn().Msgf("%s not set, defaulting to EVM_URLS", httpEnvVar)`

Reply:
@kalverra good idea. Added in 61ddd40688da4b2e171753d84dd5ca2c29ef03e2